.env

```diff
@@ -1,3 +1,3 @@
-MONGODB_CONNECTION_STRING_OLD=mongodb+srv://admin2:JiwMxK4FBOw0M4Eh@cluster0.ox05m.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
+MONGODB_CONNECTION_STRING=mongodb+srv://admin2:JiwMxK4FBOw0M4Eh@cluster0.ox05m.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```

Comment:
dont push connection string to repository, it contains youre user name and password to Atlas cluster. Move this to .env file